package.xml

```diff
@@ -38,8 +38,6 @@
   <license>New BSD</license>
   <url type="website">http://ceres-solver.org/</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
-
   <depend>atlas</depend>
   <depend>eigen</depend>
   <depend>gfortran</depend>
```

Comment:
* atlas isn't directly a dependency; what ceres really needs is liblapack and libblas, both of which atlas pulls in.  It's working in CI for us currently because libsuitesparse (which we do install) is recursively pulling in these dependencies.  However, since the ceres-solver documentation does suggest installing atlas, I'll add it as an explicit dependency to the docker image.
* Eigen is definitely a dependency, and gets installed as part of the default docker image.
* gfortran is mostly the same situation as atlas, in that it is currently getting pulled in recursively.  However, the ceres documentation doesn't explicitly mention it, and you can't install libblas without recursively pulling it in.  What should we do with that?